pkg/wave/decoder.go

```diff
@@ -9,16 +9,28 @@ import (
 )
 
 // Format represents how audio is formatted in memory
-type Format string
+type Format struct {
```

Comment:
It seems to cover only constant bit-width raw audio.
Usually it's enough but I think it's better to keep it available for other kind of formats, for example compressed data, as same as FourCC. (`wave.Audio` can be implemented for compressed data.)

Reply:
```go
type Format fmt.Stringer
```
this would be sufficient, I think.

Reply:
That's a good idea! Thanks. I've just updated the PR with that change.